src/howler.core.js

```diff
@@ -1484,6 +1535,9 @@
           // Remove any event listeners.
           sounds[i]._node.removeEventListener('error', sounds[i]._errorFn, false);
           sounds[i]._node.removeEventListener(Howler._canPlayEvent, sounds[i]._loadFn, false);
+
+          // Release the Audio object back to the pool.
+          Howler._releaseHtml5Audio(sounds[i]._node);
```

Comment:
It appears that we are releasing this regardless of if it came from the pool (say, because the pool empty when we were using `_obtainHtml5Audio` and, therefore, it makes its way back into the pool without the guarantee of it being an "enabled for mobile" object. Thoughts?